ledger/participant-integration-api/src/main/scala/platform/store/cache/EventsBuffer.scala

```diff
@@ -193,4 +208,53 @@ private[platform] object EventsBuffer {
         }
       }
   }
+
+  private[cache] def indexAfter(bufferStartInclusiveSearchResult: SearchResult): Int =
+    bufferStartInclusiveSearchResult match {
+      case InsertionPoint(insertionPoint) => insertionPoint
+      case Found(foundIndex) => foundIndex + 1
+    }
+
+  private[cache] def filterAndChunkSlice[ENTRY, FILTER_RESULT](
```

Comment:
general question about unit test coverage: do you later plan to increase unit test coverage for these nicely extracted functions?

Reply:
They are unit tested: https://github.com/digital-asset/daml/pull/13852/files#diff-2155de267cfec3ba1419d81f983111ffac90fd7a74d72a5316423ecfec89f502R289-R357